test/.gitattributes

```diff
@@ -0,0 +1,4 @@
+# Remove the text attribute from reference files, so that git doesn't convert
+# line separators on Windows machines. It causes the index files to become out
+# of sync with the fasta files.
+*.fa* -text
```

Comment:
The problem with regenerating them ourselves is that then we are testing our own output, which can lead to bugs cancelling each other out.  On the whole, I'd prefer to keep the .fai files in version control for now.  I agree that the change would be best in the top-level file.